frontend/src/components/RecordValidation.tsx

```diff
@@ -77,16 +78,6 @@ const validationColDefs: ColDef<StatusItem>[] = [
   },
 ];
 
-const defaultColDef: ColDef = {
-  wrapText: true,
-  autoHeight: true,
-  cellStyle: {
-    wordBreak: "break-word",
-    lineHeight: "1.25",
-    padding: "6px 18px",
-  },
-};
-
```

Comment:
Extracted this column definition logic that makes multi-line rows more readable & reused it in the "Historical CMO Sample Names" column.

Before:
![CleanShot 2025-05-27 at 16 40 38](https://github.com/user-attachments/assets/c6ae0e19-a266-4bd0-a33e-498494fe006b)

After:
![CleanShot 2025-05-27 at 16 42 10](https://github.com/user-attachments/assets/6067793c-2ca5-4345-b802-2cee74b92c13)